README.md

```diff
@@ -82,7 +83,7 @@ Install this project with:
 Example command:
 
 ```sh
- npx create-react-app .
+ npm install
 ```
 
```

Comment:
Great job @lRebornsl 🤝

- [ ] Kindly fix the broken issues link to make your readme file more professional.

Reply:
Solve